package/scripts/kafka.py

```diff
@@ -0,0 +1,71 @@
+# coding=utf8
+# Copyright © 2015 Cask Data, Inc.
```

Comment:
Let's say no, since it's literally a "git checkout $branch $filename" from the previous file.